scripts/import_map/update_test.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpError } from "../../src/deps.ts";
+import {
+  afterEach,
+  assertEquals,
+  assertRejects,
+  beforeAll,
+  mf,
+} from "../../src/dev_deps.ts";
+import { apiDepsIn } from "./update.ts";
+
+const depsTsMock =
+  `export { SlackAPI } from "https://deno.land/x/deno_slack_api@2.1.0/mod.ts";
+   export type {SlackAPIClient, Trigger} from "https://deno.land/x/deno_slack_api@2.2.0/types.ts";`;
```

Comment:
Good thinking to support (and test) for the case where we have two different API client versions. We shouldn't ever do this, but every time I go update the API version in the SDK I worry we're going to miss one some day 😂